src/Tizen.NUI/src/public/BaseComponents/Style/ViewStyle.cs

```diff
@@ -274,25 +313,32 @@ public Size2D MaximumSize
             set => SetValue(ClippingModeProperty, value);
         }
 
-        /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <summary>
+        /// Sets the size of a view for the width, the height and the depth.<br />
```

Comment:
consider oxford comma: "the width, the height, and the depth"

Reply:
Done.